src/content/guides/caching.md

```diff
@@ -253,13 +253,12 @@ Running another build, we would expect only our `main` bundle's hash to change,
 - The `vendor` bundle changed because its `module.id` was changed.
 - And, the `runtime` bundle changed because it now contains a reference to a new module.
 
-The first and last are expected -- it's the `vendor` hash we want to fix. Luckily, there are two plugins we can use to resolve this issue. The first is the `NamedModulesPlugin`, which will use the path to the module rather than a numerical identifier. While this plugin is useful during development for more readable output, it does take a bit longer to run. The second option is the [`HashedModuleIdsPlugin`](/plugins/hashed-module-ids-plugin), which is recommended for production builds:
+The first and last are expected, it's the `vendor` hash we want to fix. Lets use [`optimization.moduleIds`](/configuration/optimization/#optimizationmoduleids) with `'hashed'` option:
```

Comment:
~Lets~ Let's

Reply:
Great find, feel free to suggest change next time for convenience